mlir/test/Dialect/Linalg/vectorization.mlir

```diff
@@ -666,6 +666,46 @@ module attributes {transform.with_named_sequence} {
 
 // -----
 
+//  CHECK-LABEL: func @test_masked_vectorize_unit_lowpad
+func.func @test_masked_vectorize_unit_lowpad(
```

Comment:
It's not clear what makes this test unique. Could you add comments? Similar request for the other test :)